docs/guides/creatingasource.md

```diff
@@ -128,6 +147,31 @@ The message will get consumed by the source and sent directly to Sockeye in a [C
 
 ![](../assets/images/sqs-sockeye-ui.png)
 
+!!! tip "Minikube"
+    Sockeye's URL at minikube will need to be added the networking layer port.
+    ```console
+    kubectl get ksvc sockeye
+    NAME      URL                                            LATESTCREATED   LATESTREADY     READY   REASON
+    sockeye   http://sockeye.sebgoa.192.168.49.2.sslip.io   sockeye-00001   sockeye-00001   True
+    ```
+
+    Replace the network layer name and service if not using kourier.
+
+    ```console
+    kubectl get svc \
+      -n kourier-system kourier \
+      -o jsonpath='{range .spec.ports[*]}{.port}{"\t"}--> {.nodePort}{"\n"}{end}'
+    ```
+
+    Use the retrieved port for the HTTP protocol.
+    ```
+    # If the networking layer were using 31280 nodeport for HTTP
+    http://sockeye.sebgoa.192.168.49.2.sslip.io:31280/
+    ```
```

Comment:
That looks like a **lot** of complexity to me. IMO a guide should focus on the main topic, not lose readers into technical details. See my previous comment about using the alternative and very portable Service + Deployment + port-forward approach.